backend/app/views/spree/admin/shared/_head.html.erb

```diff
@@ -6,7 +6,15 @@
 <title><%= admin_page_title %></title>
 
 <%= favicon_link_tag 'favicon.ico' %>
-<%= stylesheet_link_tag 'spree/backend/all', media: 'all', data: {turbolinks_track: 'reload'} %>
+
+<% Spree::Config.admin_themes.each do |name, path| %>
```

Comment:
I'm a bit confused with this loop... Shouldn't we just render one media query, light or dark, for the active theme?

Reply:
The idea is to have a single link tag in case the theme is the same for both modes, and two link tags if they differ:

pseudo html for when it's the same theme:

```html
<link media="light, dark" src="theme1.css"/>
```

and two tags when they differ:

```html
<link media="light" src="theme1.css"/>
<link media="dark" src="theme2.css"/>
```

--- 

But I'll look into it again and see if I can simplify and ditch the loop 👍